universal-application-tool-0.0.1/app/views/applicant/ApplicantLayout.java

```diff
@@ -86,15 +89,20 @@ private ContainerTag branding() {
     return div().withId("brand-id").withClasses(ApplicantStyles.CIVIFORM_LOGO).withText("CiviForm");
   }
 
-  private ContainerTag maybeRenderTiButton(Optional<UatProfile> profile) {
+  private ContainerTag maybeRenderTiButton(Optional<UatProfile> profile, String userName) {
     if (profile.isPresent() && profile.get().getRoles().contains(Roles.ROLE_TI.toString())) {
       String tiDashboardText = "Trusted intermediary dashboard";
-      String tiDashboardLink =
-          controllers.ti.routes.TrustedIntermediaryController.dashboard().url();
-      return a(tiDashboardText)
-          .withHref(tiDashboardLink)
-          .withClasses(
-              Styles.PX_3, Styles.TEXT_SM, Styles.OPACITY_75, StyleUtils.hover(Styles.OPACITY_100));
+      String tiDashboardLink = routes.TrustedIntermediaryController.dashboard().url();
```

Comment:
Since it's only referenced once...

```suggestion
      String tiDashboardLink = controllers.ti.routes.TrustedIntermediaryController.dashboard().url();
```